docs/source/requests.md

```diff
@@ -985,6 +985,209 @@ Command to change Pool's configuration
+                                            auth_constraints: [{constraint_id: ROLE,
+                                                                role: STEWARD, 
+                                                                sig_count: 1, 
+                                                                need_to_be_owner: True},
+                                                               {constraint_id: ROLE,
+                                                                role: TRUSTEE, 
+                                                                sig_count: 1, 
+                                                                need_to_be_owner: False}
+                                                               ]
+                                           }
+                                                                 
+```
+means that change the value of node services from [VALIDATOR] to [] (demotion of node) can only TRUSTEE or STEWARD if it is owner of this transaction.
+
+- `auth_action` (enum: `ADD` or `EDIT`):
+
+    Change the rights to an action.
+    
+- `auth_type` (string):
+
+    The type of transaction to change rights to. (Example: NYM, NODE, POOL_RESTART...)
+
+- `field` (string):
+
+    Change the rights to edit(add) some values from field.
+
+- `old_value` (string; optional):
+
+    Old value of field, which can be changed to a new_value.
+
+- `new_value` (string):
+   
+   New value that can be used to fill the field.
+
+**ConstraintType:**
+
+ConstraintList
+
+- `constraint_id` (enum: `AND` or `OR`):
+
+    Type of a constraint class. Needed for determine the type of constraint for correct deserialization.
+    - `AND` logical conjunction for all constraints from `auth_constraints`
+    - `OR` logical disjunction for all constraints from `auth_constraints`
+    
+- `auth_constraints` (list of ConstraintType):
+
+    List of ConstraintType (ConstraintList or ConstraintEntity) objects
+    
+ ```
+{ 'constraint_id': 'AND',
+  'auth_constraints': [<ConstraintEntity>,
+                      <ConstraintEntity>]
+}
+```
+    
+ConstraintEntity
+
+- `constraint_id` (enum: `ROLE`):
+
+   Type of constraint, which for ConstraintEntity is always of type "ROLE". Needed for determine the type of constraint for correct deserialization.
+        
+- `role` (enum number as string; optional):
+
+    Role of a user that the NYM record is being created for. One of the following values
+
+    - None (common USER)
+    - 0 (TRUSTEE)
+    - 2 (STEWARD)
+    - 101 (TRUST_ANCHOR)
+    
+- `sig_count` (int):
+
+    The number of signatures that is needed to do some action described in the transaction fields.
+    
+- `need_to_be_owner` (boolean):
+
+    Flag to check is user must be owner of some transaction (Example: A steward must be the owner of the node to make changes to it).
+    
+- `metadata` (dict; optional):
+
+    Dictionary to additional parameters of constraint.
```

Comment:
for additional parameters of the constraint. Can be used by plugins to add additional restrictions.